templates/includes/sidebar.html

```diff
@@ -1,4 +1,11 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li> User: <a href="{% url "taxi:driver-detail" pk=user.id %}"> {{ user.username }}</a></li>
```

Comment:
Ensure that the URL name `taxi:driver-detail` follows the naming convention of using hyphens instead of underscores if applicable. Verify that this matches the URL pattern defined in `urls.py`.